### PR TITLE
Added dockerfiles,compose-files,updated readme, added env files, use of .env variables

### PR DESCRIPTION
**Pull request by MishraTanishq619**
Fixed : #89 
Fixed : #276 

Problem:
The Repository didn't had any files for Containerisation.
It takes too long to start the local server with too many steps.

Solution:
With docker and compose files, now Anyone can start development server in only one command.

Tasks: Done all the tasks
- [x]  Adding dockerfile for web ("/")
- [x]  Adding dockerfile for server ("/server")
- [x]  Adding Docker-compose file for Development server. ("Docker-Compose.Dev.Yaml")
- [x]  Adding Docker-Compose file for Production server. ("Docker-Compose.Prod.Yaml")
- [x]  Adding ".env.example" file if needed.
- [x]  Updating ReadMe.md , adding instructions to start the container.

[Screencast from 19-05-24 11:18:50 AM IST.webm](https://github.com/abhay-raj19/FitBody/assets/127047332/5f5fee05-34de-4a6b-966c-580fa2a79cef)

### CODE REVIEW

**Comment by MishraTanishq619** [comment]
Please accept my PR for GSSOC.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM node:20 as Base
+WORKDIR /app
+COPY package*.json  ./
```
Comment (abhay-raj19) [inline Dockerfile:3]: No need for `package*.json`
you can simply mention up the package.json , what I think so

**Comment by abhay-raj19** [comment]
Changes not looks good to me simply add up the dockerfile and compose file for that and add the section how to start it with docker in readme.md don't change up the readme fully it.

File: Dockerfile (hunk 1 above)
Comment (MishraTanishq619) [reply Dockerfile:3]: package-lock.json - will be not included then..
That was fine.

**Pull request closed without merging**